learning-team/2020-04/2020-04-30.md

```diff
@@ -19,20 +17,46 @@ Add yourself to the list if you attend and check the box!
 
 ### Guests
 
-## Weekly Agenda
+Add yourself to the list if you attend and check the box:
+
+- [ ]
 
-- [ ]  Review action items (update weekly)
-- [ ]  Review [Initiatives](https://github.com/orgs/ember-learn/projects/19) [Board](https://github.com/orgs/ember-learn/projects/33) [](https://github.com/orgs/ember-learn/projects/19)(every second week)
-- [ ]  How is our PR queue doing? How many older than a week? [Tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
-- [ ]  Review new RFCs/[RFC Tracking](https://github.com/emberjs/rfc-tracking)
-- [ ]  Any suggestions for the Ember Times? Add writeup ideas / call for contributors / suggestions to [the latest PR](https://github.com/ember-learn/ember-blog/pulls?q=is%3Aopen+is%3Apr+label%3A%22%F0%9F%97%9E+embertimes%22%20or%20#support-ember-times) or [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on Discord.
+## Topics - All-hands
 
-## TODOS
+### Review action items (weekly)
+See: https://github.com/orgs/ember-learn/projects/47.
 
+### Recurring - Review Initiatives (every second week)
+See: https://github.com/orgs/ember-learn/projects/19
 
-## Topics
+### Review Issue tracking sheet (weekly)
+See: https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454
 
-### Topic goes here (initials)
+### ADD YOUR TOPIC HERE (initials)
 
 ## Any Questions?
 Questions, comments or concerns? Submit a comment or PR for this set of notes.
+
+### Any suggestions for the Ember Times? (weekly)
+Any suggestions for the Ember Times? Add writeup ideas / call for contributors / suggestions to [the latest PR](https://github.com/ember-learn/ember-blog/pulls?q=is%3Aopen+is%3Apr+label%3A%22%F0%9F%97%9E+embertimes%22%20or%20#support-ember-times) or [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on Discord.
+
+## Topics - Breakout - Infrastructure/Tooling
+
+### 404 errors on API docs
+
+## Topics - Breakout - Guides/API docs/Tutorials
+
+### Define criteria for which content should be published to Guides and which should be published to CLI Guides
```

Comment:
```suggestion
### Define criteria for which content should be published to Guides and which should be published to CLI Guides

See [this issue](https://github.com/ember-learn/cli-guides/issues/168) proposing a rule of thumb.
See [this PR](https://github.com/ember-learn/cli-guides/pull/163) for an example thing to solve

RM: There are 2 problems to solve, what to do with new content and what to do with old content.
Another rule of thumb is that the Ember Guides should aim to cover the 80% of users experience.

Summary: For new content, we think it makes sense to focus on build time content for the CLI, and for Ember Guides, it should focus on runtime and the 80% (general) user needs.

What to do with old content that we want to move?

JW: This is a somewhat new thing to solve because the CLI guides used to be stagnant.

RM: For best UX, we should have a short paragraph that summarizes and links to the new content home.
Redirecting would be jarring.

JG: I like this idea - including multiple resources and info so that the guides do not seem like a pruned tree.

JW: What would be the effects if we leave build targets where it is?

RM: Would like us to more deeply document this in the CLI guides and so moving it helps with that. CLI guides needs more content and this is good to put there. Otherwise this is more content to maintain in the context of the "general" Ember experience. We don't currently have multiple builds but we would have to revisit if we do.
The Ember CLI guides are very attached to the `ember new blueprint`.

JW: The CLI guides by necessity are about a lot more than just the CLI commands

JW: What do we need to do to move forward?

RM: notify the team. Can always revert if we need to since we are not removing the page.

Summary - ok to move content, the existing Build Targets page as long as we have a sister PR that points to the new place and provides some rich content.

### How do we want to use the subgroup time?

JD: interested to help move the guides along! Co-writing the typescript guides and want to keep them in harmony

JG: The guides are most accessible to people who are new to Ember. They were important to my experience and so I like to help provide this experience to someone else.

JW: Want to always be removing barriers for new people, regardless of the project I am working on

RM: Make Ember easier to learn for anyone! Broaden the user base as much as possible.

### Guides issues

JW: If you could wave a magic wand and one issue is done, what would it be? We have people willing to help so this is not about what you would want to work on, just general wishes for the project.

JW: https://github.com/ember-learn/guides-source/issues/1320 and https://github.com/ember-learn/guides-source/issues/20

RM: Need to triage the Issue queue

JG: I mostly just see small issues.

JW: We should think then about the things we see as areas of improvement and open issues for them.

RM: We need to bring templates back!
```